### ISSUE TITLE
Change of accessible name within a Live Region is output inconsistently

### ISSUE THREAD

**Issue by JAWS-test** [FreedomScientific/VFO-standards-support#274]
### Summary

Change of accessible name within a Live Region is output inconsistently

1. Save as HTML file:
```
<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="utf-8">
		<title>live region</title>
	</head>
	<body>
		<div role=status>
			<button aria-label=button1 id=1>x</button>
			<button aria-labelledby=3a id=2>y</button>
			<button aria-labelledby=3b>z</button>
		</div>
		<span id=3a>button2</span>
		<span id=3b>button3</span>
		<span id=4>new aria-labelledby1</span>
		<button onClick="document.getElementById('1').setAttribute('aria-label','new aria-label');">Change 1st button</button>
		<button onClick="document.getElementById('2').setAttribute('aria-labelledby','4');">Change 2nd button</button>
		<button onClick="document.getElementById('3b').textContent='new aria-labelledby2';">Change 3rd button</button>
	</body>
</html> 
```
2. Activate the buttons "Change 1st/2nd/3rd button"

### Expected result
* I would expect: Change of accessible name of the buttons "x", "y" and "z" is output correctly
* But https://rawgit.com/w3c/aria/master/#aria-relevant says: no ouput because no change of text node

### Actual result
* Chrome: output with all buttons
* IE 11: no output at all
* Firefox: output with first and second button, but not with third

See also https://github.com/w3c/aria-practices/issues/78 and many other problems with live regions: https://github.com/FreedomScientific/VFO-standards-support/issues?q=live+region

#### JAWS version and build number

JAWS 2019.1907.42
 
#### Operating System and version

Windows 8

#### Browser and version:

Chrome 76.0.3809.100
Firefox 68.0.2
Internet Explorer 11.0.9600.19431

**Comment by lamador** [comment]
The following results were found when using the latest version of JAWS (2020.2006.12), Chrome (84.0.4147.89), Firefox (78.0.2), and IE11 (11.959.18362.0) on Windows 10:
- JAWS + Firefox: Change in accessible name of one button is announced upon activating the "Change 1st button" or "Change 2nd button" `button` elements, but not the "Change 3rd button".
- JAWS + Chrome/IE11: No announcements were made when activating any of the "Change" `button` elements.

For comparison, NVDA (version 2020.1) was also test with the same group of browsers and obtained the following results:
- NVDA + Firefox: Change in accessible name of one button is announced upon activating the "Change 1st button" or "Change 2nd button" `button` elements, but not the "Change 3rd button".
- NVDA + Chrome: Visible labels of "x", "y", and "z" buttons announced when any of the "Change" `button` elements activated.
- NVDA+ IE11: No announcements were made when activating any of the "Change" `button` elements.

The purpose of a live region is to convey messages or changes in content not visible or readily perceivable by users. The implementation of the status role in the provided test case to convey a change in the accessible name of a button is arguably not the appropriate use of a live region as the test results show. The lack of an announcement by JAWS when paired with Chrome or IE11 should be the expected behavior (rather than the announcement when using JAWS with Firefox) when the first and second "Change" buttons are activated since the visible text of the "x", "y", and "z" buttons aren't updated.  Additionally, the lack of announcement when activating "Change 3rd button" is the correct behavior since the change in text is not contained within a live region.

Additional testing was conducted on updating button text instead of the aria-label using sample code in [CodePen #274](https://cdpn.io/TPG/debug/KKVJegP). The results are basically the same as the original test. 
- JAWS + Firefox/IE11, NVDA + Firefox: Announcement of changed button text, but text of other buttons not announced
- JAWS + Chrome, NVDA + IE11: No announcement
- NVDA + Chrome: Visible labels of "x", "y", and "z" buttons announced when any of the "Change" `button` elements activated.

Note that in the aforementioned code sample, another live region was added for the plain text to demonstrate the proper use of a live region. This text is updated when activating the "Change 3rd button" `button` element. Consistent announcements of the updated text were made by all the browser and screen reader combinations previously mentioned.

In summary, improperly using the status role is the cause of the announcement issues identified rather than with JAWS.
